File: docs/src/testers/scraping/link_placeholders.py
# @sniptest filename=link_placeholders.py
# @sniptest show=6-7
from notte_sdk import NotteClient

client = NotteClient()
url = "https://example.com"

# Use placeholders for links and images
markdown = client.scrape(url, use_link_placeholders=True)

results = [
    isinstance(markdown, str),
    "Example Domain" in markdown,
    "[Learn more](link1)" in markdown,
    "iana.org" not in markdown,
]

File: docs/src/testers/scraping/link_placeholders.ts
// @sniptest filename=link_placeholders.ts
// @sniptest show=6-9
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();
const url = 'https://example.com';

// Use placeholders for links and images
const markdown = await client.scrape(url, {
  use_link_placeholders: true,
});

export const results = [
  typeof markdown === 'string',
  markdown.includes('Example Domain'),
  markdown.includes('[Learn more](link1)'),
  !markdown.includes('iana.org'),
];
